### ISSUE TITLE
`antfu/consistent-list-newline` autofix breaks between items, in node ArrowFunctionExpression

### ISSUE THREAD

**Issue by pengzhanbo** [antfu/eslint-plugin-antfu#16]
### Describe the bug

**before**
```tsx
function TodoList() {
  const { data, isLoading } = useSwrInfinite(
    (page) => ['/api/todo/list', { page }],
    ([, params]) => getToDoList(params),
  )
  return <div></div>
}
// error  Should have line breaks between items, in node ArrowFunctionExpression  antfu/consistent-list-newline
```

**Autofix result:**
```tsx
function TodoList() {
  const { data, isLoading } = useSwrInfinite(
    page
    => ['/api/todo/list', { page }],
//  ^^^  Unexpected line breaks
    ([, params]) => getToDoList(params),
  )
  return <div></div>
}
```
Unexpected line breaks that do not meet expectations.

### Reproduction

Reproduction

### System Info

```Shell
eslint-plugin-antfu@1.0.12
```

### Used Package Manager

pnpm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/antfu/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by antfu** [comment]
I am not able to reproduce it: 0fe40618910227d98a84bfb58e32ab2e99b23d01

**Comment by pengzhanbo** [comment]
This bug was fixed in `v1.0.13` .
